src/ledger/LedgerTxnOfferSQL.cpp

```diff
@@ -648,9 +646,9 @@ class BulkLoadOffersOperation
         for (auto const& k : keys)
         {
             assert(k.type() == OFFER);
-            if (k.offer().offerID <= INT64_MAX)
+            if (k.offer().offerID > 0)
```

Comment:
This isn't consistent with line 23, which includes queries with `offerID == 0`.

Reply:
Updated for consistency.